rdmo/management/viewsets.py

```diff
@@ -46,12 +46,13 @@ def create(self, request, *args, **kwargs):
             # step 2-6: parse xml, validate and convert to
             xml_parsed_elements, errors = parse_xml_to_elements(xml_file=import_tmpfile_name)
         except ValidationError as e:
-            logger.info(f'Import failed with XML parsing errors. {", ".join(map(str, errors))}')
+            logger.info('Import failed with XML parsing errors.')
```

Comment:
There is this part `{", ".join(map(str, errors))}'` missing now?

Reply:
yes I've removed that part there. Think it was a mistake to use `errors` there because it wouldn't even be defined when that exception occurs.